apps/ledger-live-mobile/src/screens/PTX/BuyAndSell/index.tsx

```diff
@@ -40,11 +40,15 @@ export function BuyAndSellScreen({ route }: Props) {
    * to ensure the context is reset. last-screen is used to give an external app's webview context
    * of the last screen the user was on before navigating to the external app screen.
    */
-  if (manifest?.id && INTERNAL_APP_IDS.includes(manifest.id)) {
-    AsyncStorage.removeItem("last-screen");
-    AsyncStorage.removeItem("manifest-id");
-    AsyncStorage.removeItem("flow-name");
-  }
+  useEffect(() => {
+    (async () => {
+      if (manifest?.id && INTERNAL_APP_IDS.includes(manifest.id)) {
+        await AsyncStorage.removeItem("last-screen");
+        await AsyncStorage.removeItem("manifest-id");
+        await AsyncStorage.removeItem("flow-name");
+      }
+    })();
+  });
```

Comment:
I think we will need a dependency array here? 
```suggestion
  useEffect(() => {
    (async () => {
      if (manifest?.id && INTERNAL_APP_IDS.includes(manifest.id)) {
        await AsyncStorage.removeItem("last-screen");
        await AsyncStorage.removeItem("manifest-id");
        await AsyncStorage.removeItem("flow-name");
      }
    })();
  }, []);
```

Reply:
yup, eslint had issues, added the disable.